### PR TITLE
feat(Faq): faq component created

### PR DESCRIPTION
**Pull request by zKriguer**
This commits changes, are creating a component to answer frequent **asked questions**

image

### CODE REVIEW

File: src/components/constants/faq.ts
Comment (fellipeutaka) [inline src/components/constants/faq.ts:1]: the constants folder was created inside the components folder

File: src/lib/utils.ts
Comment (fellipeutaka) [inline src/lib/utils.ts:1]: unnecessary

**Review by fellipeutaka — APPROVED** [review]

**Review by MatheusLukas — APPROVED** [review]
Muito bom

File: package.json
[the remark this answers is not in the document]
Comment (zKriguer) [reply package.json:1]: resolved on next pull request

**Pull request merged**